server/index.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const ejs = require('ejs');
+const bp = require('body-parser');
+
+const app = express();
+
+app.use(express.static('public'));
+app.use(bp.json());
+app.engine('html', ejs.renderFile);
+app.get('/:id', (req, res) => {
+  res.render('../public/index.html');
+});
+app.set('view engine', 'html');
+
+const port = process.env.PORT || 4000;
```

Comment:
Is this 4000 the selected port number because of a reference found within docs or other instructions? Or is this where we would all use our designated proxy port assignments?

![Screen Shot 2020-02-04 at 6 20 27 PM](https://user-images.githubusercontent.com/54820169/73805327-42790b80-477b-11ea-91bc-c5ccd6430109.png)

Reply:
Yes, It is just something different from the list on your screenshot